tests/typescript_conditional_types/jsfmt.spec.js

```diff
@@ -0,0 +1 @@
+run_spec(__dirname, ["typescript"]);
```

Comment:
Can babylon parse this too?

Reply:
Pretty sure we don't atm

Reply:
Update: PR support in Babel -> https://github.com/babel/babel/pull/7404

Reply:
@existentialism Yep, Babel 7 supports TypeScript already. I prefer current method though because I'm not sure if the syntax support is always up-to-date for Babel.